dotnet-tools.json

```diff
@@ -13,6 +13,12 @@
       "commands": [
         "dotnet-format"
       ]
+    },
+    "dafny": {
+      "version": "3.7.1.40621",
```

Comment:
Is this intended to be a fixed version number, or should there be a mechanism to keep it up-to-date with the latest release (or whatever's in the repo)?

Reply:
No, it has to be a fixed number to avoid recursion on the same version (i.e. you break the dafny compiler, and because you have compiler code that is written in Dafny, you cannot compile it anymore using the current branch).